posthog/settings/data_stores.py

```diff
@@ -288,7 +288,7 @@ def _parse_kafka_hosts(hosts_string: str) -> List[str]:
 
 CACHES = {
     "default": {
-        "BACKEND": "django_redis.cache.RedisCache",
+        "BACKEND": "django.core.cache.backends.dummy.DummyCache",
```

Comment:
🤔

Reply:
Oh poop, none of these were meant to get committed 🤦  thank you

Reply:
Now that's some foul language 🤣

Reply:
All fixed